server/askserver/askserver/auth_backends.py

```diff
@@ -0,0 +1,26 @@
+from django.contrib.auth.backends import BaseBackend
+import requests
+
+class OVF20AuthBackend(BaseBackend):
+    def authenticate(self, request, username=None, password=None):
+        # Make a request to the OVF 2.0 authentication server
+        response = requests.post('https://ovf2.0/authenticate', data={'username': username, 'password': password})
```

Comment:
is `https://ovf2.0/authenticate` a dummy value? where will it point in prod? e.g. google authentication (if UNCC Duo is out of the question at the 11th hour)

Reply:
It's a placeholder for the endpoint/ API endpoint